Docs/source/developers/performance_tests.rst

```diff
@@ -0,0 +1,108 @@
+.. _developers-performance_tests:
+
+Automated performance tests
+===========================
+
+WarpX has automated performance test scripts, which run weak scalings for various tests on a weekly basis. The results are stored in the `perf_logs repo <https://github.com/ECP-WarpX/perf_logs>`_ and plots of the performance history can be found on `this page <https://ecp-warpx.github.io/perf_logs/>`_.
+
+These performance tests run automatically, so they need to do ``git`` operations etc. For this reason, they need a separate clone of the source repos, so they don't conflict with one's usual operations. This is typically in a sub-directory in the ``$HOME``, with variable ``$AUTOMATED_PERF_TESTS`` pointing to it. Similarly, a directory is needed to run the simulations and store the results. By default, it is ``$SCRATCH/performance_warpx``.
+
+The test runs a weak scaling (1,2,8,64,256,512 nodes) for 6 different tests ``Tools/PerformanceTests/automated_test_{1,2,3,4,5,6}_*``, gathered in 1 batch job per number of nodes to avoid submitting too many jobs.
+
+Setup on Summit @ OLCF
+----------------------
+
+Here's my setup on Summit, feel free to adapt it to your needs:
+
+.. code-block:: sh
+
+   # I put the next three lines in $HOME/my_bashrc.sh
+   export proj=aph114  # project for job submission
```

Comment:
```suggestion
   export proj=<PorjectID>  # project for job submission
```
I thought we always tried to use the projectID (for privacy), like you did in the libEnsemble example in Tools.
Is it not important?

Reply:
This page is meant to be used by WarpX developers, so I would be OK to hard-code the project name for now.